Update changelog for 2.0.0-preview.2 release (#682)
twilio/twilio-voice-react-native: CHANGELOG.md

> 2.0.0-preview.2 (April 29, 2026)
> ================================
> 
> ## Features
> 
> - Added support for configuring custom ICE servers and ICE transport policy when creating outgoing calls using `Voice.connect` via the new `iceServers` and `iceTransportPolicy` options.
> 
>   *Note*: Custom ICE server and ICE transport policy functionality for incoming calls will come in a future version.
> 
> ## Changes
> 
> - Updated the native Twilio Voice iOS SDK and Twilio Voice Android SDK dependencies.

Reviewer: ```suggestion
2.0.0-preview.2 (April 29, 2026)
================================

## Features

- Added support for custom ICE servers and ICE transport policy for outgoing calls initiated with `Voice.connect`, via the new `iceServers` and `iceTransportPolicy` options.

  *Note*: This currently applies only to outgoing calls. Support for incoming calls will be added in a future version.

## Changes

- Updated the native Twilio Voice iOS SDK and Twilio Voice Android SDK dependencies.
```

Author: Added, but dropped the recommended comma. Would be grammatically incorrect to add it.

Reviewer: Weird. ChatGPT says otherwise 🤔  But no big deal. +1

Author: Oh, maybe I'm wrong then. It felt like the flow got kind of wonky with the comma included. Ok, let's just merge it then, no big deal with or without it. Thanks!

2.0.0-preview.2 (April 29, 2026)

> ================================

Author: The length is fine.

## Features

- Added support for custom ICE servers and ICE transport policy for outgoing calls initiated with `Voice.connect` via the new `iceServers` and `iceTransportPolicy` options.

>   *Note*: This currently applies only to outgoing calls. Support for incoming calls will be added in a future version.

Author: Tense of changelog entries is not a big deal.
